docs/pages/guides/webhooks.mdx

```diff
@@ -5,6 +5,15 @@ meta:
   description: "Learn Liveblocks Webhooks concepts and implementation"
 ---
 
+<Banner title="Open beta">
+
+Webhooks are currently in open beta. If you have any questions or feedback,
+please reach out to us in the dedicated discord channel,
```

Comment:
I beleive we cap Discord (at least we do in other places)- correct me if I'm wrong

```suggestion
please reach out to us in the dedicated Discord channel,
```